ballerina/tests/batch-execute-query-test.bal

```diff
@@ -96,6 +100,21 @@ function batchInsertIntoDataTableFailure3() {
     test:assertTrue(result is ApplicationError);
 }
 
+@test:Config {
+    groups: ["batch-execute"]
+}
+function batchInsertIntoDataTable4() returns error? {
+    string path = check file:getAbsolutePath("tests/resources/sample/parameterized_query.bal");
+    Process process = check exec("bal", {}, (), "run", path);
+    int waitForExit = check process.waitForExit();
+    int exitCode = check process.exitCode();
+    io:ReadableByteChannel readableResult = process.stderr();
+    io:ReadableCharacterChannel sc = new (readableResult, "UTF-8");
+    string outText = check sc.read(100000);
+    string[] texts = regex:split(outText, "\n");
+    test:assertTrue(strings:includes(texts[3], "method is too large"));
```

Comment:
Is this a negative test?

Reply:
Yes, this is a negative test. I have added a `negative` suffix in c0db6a8.

Reply:
Do we need a negative test here? Out object is this error should not be thrown right?

Reply:
Changed to positive test in 2406fa6